framework/include/lw_aifw/AIServiceCore.h

```diff
@@ -21,8 +21,8 @@
 #include <signal.h>
 #include "lw_aifw/lw_aifw.h"
 #include "lw_aifw/AIDataSource.h"
-#include "AIDataBuffer.h"
-#include "AIModel.h"
+#include "../../src/lw_aifw/include/AIDataBuffer.h"
```

Comment:
It should be synced with prod branch, currently they're different

Reply:
AI Framework has public interfaces at framework/include/lw_aifw.
Public interfaces use lw_aifw implementation specific header files at location framework/src/lw_aifw/include.
Sample application is made in CPP & includes AIServiceCore.h.
Referring to suggestion https://github.com/Samsung/TizenRT/pull/5788#discussion_r1147397427, I also think application should not directly include non public header files.

Reply:
I think we should clean header files.
Public interface should not refer to private interface.
In other words, header in fw/include should not refer to fw/src/include, on the contrary, it acceptable.